argparse/__init__.py

```diff
@@ -65,9 +65,26 @@ def load_conflicting_package(name, not_name, local_module):
 tools = []
 
 
+def get_arg2cwl_parser():
+    help_text = """
+    argparse2cwl forms CWL command line tool from Python tool
+    """
+    arg2cwl_parser = ap.ArgumentParser(prog=sys.argv[0], description=help_text, add_help=False)
+    arg2cwl_parser.add_argument('tools', nargs='*',
+                                help='Command for running your tool(s), without arguments')
```

Comment:
Is this a required option or does it override the default?

I would give an example: `python -m modulename` or `my_script`